runtime/compiler/runtime/SymbolValidationManager.hpp

```diff
@@ -637,9 +641,9 @@ class SymbolValidationManager
 
    struct SystemClassNotWorthRemembering
       {
-      const char * const _className;
+      char *_className;
```

Comment:
Can't we leave the first `const` here. The second one refers to the `_className` being immutable, but the first one refers to the string of characters being immutable.

Reply:
True. I can push the change once the tests pass.

Reply:
The tests on Power take very long for some reason, but on x86 they passed. You could do this change now.